src/server/game/Entities/Unit/Unit.h

```diff
@@ -885,12 +885,26 @@ enum ReactiveType
 
 #define MAX_GAMEOBJECT_SLOT 4
 
-enum PlayerTotemType
+enum PlayerTotemCategory
 {
-    SUMMON_TYPE_TOTEM_FIRE  = 63,
-    SUMMON_TYPE_TOTEM_EARTH = 81,
-    SUMMON_TYPE_TOTEM_WATER = 82,
-    SUMMON_TYPE_TOTEM_AIR   = 83
```

Comment:
are these old values not longer used? they are still in db2

Reply:
Only SUMMON_TYPE_TOTEM_WATER are used now for player, after others spells are for creature etc.. so don't need for get the model because it's another stuff